src/components/menu/types.d.ts

```diff
@@ -6,7 +6,11 @@ export type MenuItemProps = {
   onPress: () => void;
   isTitle?: boolean;
   isDestructive?: boolean;
+  /**
+   * @deprecated typo here for compatibility
+   */
```

Comment:
I kept the old field so this is not a breaking change.

Reply:
This is exactly well thought update but I removed it, package is already very early stage and not used for many people now. Will continue with other typo fixes 🙌🏽